pkg/qliksense/context_configs.go

```diff
@@ -106,7 +102,8 @@ func (q *Qliksense) processSecret(ra *api.ServiceKeyValue, rsaPublicKey *rsa.Pub
 		if k8sSecret.Data == nil {
 			k8sSecret.Data = map[string][]byte{}
 		}
-		k8sSecret.Data[ra.Key] = []byte(base64EncodedSecret)
+		// v1.Secret does enconding, so no need to encode again
```

Comment:
@ashwathishiva did you remember double encoding issue

Reply:
yes, I do